### DISCUSSION TITLE
Null object as expander in xaml

### DISCUSSION THREAD

**Discussion by Sonderwoods** [picoe/Eto#2530] [Q&A] [answered]
Hi,

When generating an UI with code we can add null objects as expanding/stretching voids.

How are these created using the xeto/xaml code?

**Comment by cwensley** [comment, marked as answer]
For StackLayout: `<StackLayoutItem Expand="True"/>`
For TableLayout: `<TableRow ScaleHeight="True"/>` or `<TableCell ScaleWidth="True"/>`
For DynamicLayout: `<DynamicControl XScale="True"/>` or `<DynamicControl YScale="True"/>`

Hope this helps!
